scripts/status_map.js

```diff
@@ -28,7 +28,7 @@ let shieldImageURLs = Object.fromEntries(
     `https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/${sprite}.svg`,
   ])
 );
```

Comment:
`shieldImageURLs` is unused and can be removed safely. This script only cares about the keys of `shields` anyways.

Reply:
Fixed in 9bd2acf